### PR TITLE
docs: create security policy

### LINKED ISSUE #4369: Report a security vulnerability

### PR DESCRIPTION
**Pull request by q9f**
## Description

Fix missing security policy 

## Type of change

Docs: add security policy

Closes #4369

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+- :hourglass_flowing_sand: Web3.js currently undergoes a quick release cycle with regular minor and patch releases.
+- :globe_with_meridians: We encourage using the [most recently released version](https://github.com/ChainSafe/lodestar/releases/latest) from the `v1.x` release track before reporting an issue.
+
+Future rewrites and feature tracks `v2.x`, `v3.x`, and  `v4.x`:
+- :hourglass: Web3.js currently undergoes a complete refactoring and rewrite. At this time it is not recommended to use any of these release tracks, yet.
+- :stop_sign: We encourage **not** using these releases until official stable releases will be announced.
+
+## Reporting a Vulnerability
+- :rotating_light: Please, send vulnerability reports to `security@chainsafe.io`.
+- :warning: **Please do not file a public ticket** mentioning the vulnerability, as doing so could increase the likelihood of the vulnerability being used before a fix has been created, released and installed on the network.
```
Comment (nazarhussain) [inline SECURITY.md]:
```suggestion
- :warning: **Please do not file a public ticket** mentioning the vulnerability, as doing so could increase the likelihood of the vulnerability being exploited before a fix has been created, released and installed on the network.
```

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Security Policy
+## Supported Versions
+Legacy-compatibility releases `v1.x`:
+- :hourglass_flowing_sand: Web3.js currently undergoes a quick release cycle with regular minor and patch releases.
+- :globe_with_meridians: We encourage using the [most recently released version](https://github.com/ChainSafe/lodestar/releases/latest) from the `v1.x` release track before reporting an issue.
+
+Future rewrites and feature tracks `v2.x`, `v3.x`, and  `v4.x`:
+- :hourglass: Web3.js currently undergoes a complete refactoring and rewrite. At this time it is not recommended to use any of these release tracks, yet.
```
Comment (nazarhussain) [inline SECURITY.md:8]: Does the term rewrite only valid for `4.x` branch?

**Review by nazarhussain — CHANGES_REQUESTED** [review]

File: SECURITY.md (hunk 2 above)
Comment (LuaxY) [reply SECURITY.md:8]: The link redirect to `lodestar` releases page and not `web3.js`, is that normal ?

Comment (q9f) [reply SECURITY.md:8]: yeah, that's normal when you submit PRs before first coffee 😆

Comment (philknows) [reply SECURITY.md:8]: > Does the term rewrite only valid for `4.x` branch?

That's generally how we've been using that term. rewrite = 4.x

**Review by Spacesai1or — APPROVED** [review]

**Comment by Spacesai1or** [comment]
#4369 should link to this

**Review by nazarhussain — APPROVED** [review]

**Pull request merged**